### PR TITLE
Show total no. of strands and strand groups created in the strand dump

### LINKED ISSUE #37655: [Improvement]: Show total no. of strands and strand groups created in the strand dump
### Description

$title

### Describe your problem(s)

_No response_

### Describe your solution(s)

_No response_

### Labels

jBallerina

### Related issue(s) (optional)

_No response_

### Suggested label(s) (optional)

_No response_

### Suggested assignee(s) (optional)

_No response_

### PR DESCRIPTION
**Pull request by Nadeeshan96**
$title
Fixes #37655

```bal
public class MockListener {
    public function 'start() returns error? {
    }

    public function gracefulStop() returns error? {
    }

    public function immediateStop() returns error? {
    }

    public function attach(service object {} s, string[]|string? name = ()) returns error? {
    }

    public function detach(service object {} s) returns error? {
    }
}

public listener MockListener mockListener0 = new ();

public function main() {}
```
For the above bal code if we take the strand dump in the listener phase, it is given as follows.
```
Ballerina Strand Dump [2022/09/14 18:49:48]
===========================================

Total strand group count        :       5
Total strand count              :       3
Active strand group count       :       0
Active strand count             :       0

===========================================
```
3 strands created are for `.<init>`, `main` and `start` functions. When `SIGINT` signal is sent to shutdown the program another 23 `stop` strands are created.

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Review by HindujaB — APPROVED** [review]
LGTM

File: bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/scheduling/SchedulerItem.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+package io.ballerina.runtime.internal.scheduling;
+
+import io.ballerina.runtime.internal.values.FutureValue;
+
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+/**
+ * Represent an executable item in Scheduler.
```
Comment (HindujaB) [inline bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/scheduling/SchedulerItem.java:27]:
```suggestion
 * Represents an executable item in Scheduler.
```

File: bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/scheduling/ItemGroup.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2019, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
```
Comment (warunalakshitha) [inline bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/scheduling/ItemGroup.java:2]: shall we change to 2019-2022

File: bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/scheduling/SchedulerItem.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2019, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
```
Comment (warunalakshitha) [inline bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/scheduling/SchedulerItem.java:2]: 2019-2022

**Review by warunalakshitha — APPROVED** [review]
LGTM

**Pull request merged**